flex/07-flex-layout-2/README.md

```diff
@@ -20,6 +20,8 @@ On a smaller screen it will look like this:
 
 ![smaller](./desired-outcome-smaller.png)
 
+Note: The emojis may instead show up as one or several text symbols (e.g. &#9734;&#9794;) if the Noto Color Emoji font family is not properly installed. This does not affect the exercise and can be ignored.
```

Comment:
Just to make it a little less specific about that particular font family, what do you think about:

```suggestion
Note: The emojis may instead show up as one or several text symbols (e.g. &#9734;&#9794; if you don't have an emoji based font family installed on your operating system. This does not affect the exercise and can be ignored.
```

Reply:
Makes sense, I've included the change - thanks for reviewing!